dll/win32/browseui/aclistisf.h

```diff
@@ -25,40 +26,68 @@ class CACListISF :
     public CComObjectRootEx<CComMultiThreadModelNoCS>,
     public IEnumString,
     public IACList2,
+    public ICurrentWorkingDirectory,
     public IShellService,
     public IPersistFolder
 {
 private:
+    enum LOCATION_TYPE
+    {
+        LT_DIRECTORY,
+        LT_DESKTOP,
+        LT_MYCOMPUTER,
+        LT_FAVORITES,
+        LT_MAX
+    };
+
     DWORD m_dwOptions;
+    INT m_iNextLocation; /* LOCATION_TYPE */
```

Comment:
Why INT here, and not directly LOCATION_TYPE?

Reply:
OK.